suite/tests/CMakeLists.txt

```diff
@@ -176,6 +176,11 @@ else ()
   set(SKIP_LESS_DIFFERENTIATED_TESTS_FOR_PULL_REQUEST OFF)
 endif ()
 
+# i#4800: Run long test suite on push-to-master events.
+if (RUN_LONG_SUITE OR ("$ENV{CI_TRIGGER}" STREQUAL "push" AND
+                       "$ENV{CI_BRANCH}" STREQUAL "refs/heads/master"))
```

Comment:
(Vs the #4059 check which is for !pull_request: I suppose we don't want the long suite on pushes to project or feature branches so it makes sense to be different.)

Reply:
Right, the current condition works fine if we don't want long suite on pushes to project or feature branches. 

About running long suite on pushes to project branches only: this may help in verifying if things are ok after each reviewed PR is pushed to the project branch, or else one may see a bunch of long suite failures when the project branch is finally merged to master. If we did want that, I wonder how we'd do it. Commits to feature and project branches both is seen as a push event by Github I think. Not sure how'd we differentiate, so that it triggers on the latter only. 

If that's not needed now, we can let this be as-is.

Reply:
I would just leave it as-is.